crates/oxc_linter/src/rules/eslint/no_useless_escape.rs

```diff
@@ -152,11 +152,8 @@ impl Rule for NoUselessEscape {
     }
 }
 
-fn is_within_jsx_attribute_item(id: NodeId, ctx: &LintContext) -> bool {
-    if matches!(ctx.nodes().parent_kind(id), Some(AstKind::JSXAttributeItem(_))) {
-        return true;
-    }
-    false
+fn is_within_jsx_attribute(id: NodeId, ctx: &LintContext) -> bool {
+    matches!(ctx.nodes().parent_kind(id), Some(AstKind::JSXAttribute(_)))
```

Comment:
Same as previous comment.

Reply:
good catch. i've checked and this change is ok